### PR TITLE
Learner dashboard

### PR DESCRIPTION
**Pull request by mbareta**

### CODE REVIEW

File: themes/fast-track-theme/lms/templates/courseware/info.html [hunk 1]
```diff
@@ -78,57 +141,85 @@ <h2 class="page-subtitle">${course.display_name_with_default}</h2>
...
-          % if last_viewed_item:
-            <a href=${'/courses/' + last_viewed_item['course_id'] + '/jump_to/' + last_viewed_item['module_id']}>
-              <h3>${last_viewed_item['object'].display_name}</h3>
-              ${last_viewed_item['modified'].strftime("%B %d, %Y")}
-            </a>
+          <div id="tab-tools" class="ft-tab">
+            <iframe src="http://localhost:3000/lti/deliverables/" frameborder="0" width="100%" height="300px"></iframe>
+          </div>
+
+          <script>
```
Comment (mbareta) [inline themes/fast-track-theme/lms/templates/courseware/info.html:205]: I have no intention to extract this to a JS file.

Comment (vukoviciv) [reply themes/fast-track-theme/lms/templates/courseware/info.html:205]: that's the spirit!

File: themes/fast-track-theme/lms/static/sass/lms-main-v1.scss [hunk 2]
```diff
@@ -121,3 +123,313 @@ $link-color: #6f6f6f;
...
+
+.sidebar-ft {
+  width: 20%;
+  margin-left: 5%;
+  display: inline-block;
+  vertical-align: top;
+  color: #777;
+
+  .home {
+    border: 1px solid #ddd;
```
Comment (MiroDojkic) [inline themes/fast-track-theme/lms/static/sass/lms-main-v1.scss:161]: This is getting overwritten by border: 0 none; below

File: themes/fast-track-theme/lms/static/sass/lms-main-v1.scss [hunk 3]
```diff
@@ -121,3 +123,313 @@ $link-color: #6f6f6f;
...
+    color: #aaa;
+    padding: 10px;
+  }
+}
+
+.search-box {
+  background: url("../images/newSearchBg.jpg");
+  border: 1px solid #ddd;
+  margin-left: -3px;
+  border-left: 0 none;
```
Comment (MiroDojkic) [inline themes/fast-track-theme/lms/static/sass/lms-main-v1.scss:236]: Shouldn't just none or 0 be enough to remove it?

**Pull request merged**